Directory.Packages.props

```diff
@@ -9,6 +9,8 @@
     <PackageVersion Include="BenchmarkDotNet" Version="0.13.5" />
     <PackageVersion Include="FluentAssertions" Version="6.11.0" />
     <PackageVersion Include="GitHubActionsTestLogger" Version="2.3.2" />
+    <PackageVersion Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+    <PackageVersion Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```

Comment:
These packages are deprecated - we should add [a framework reference](https://learn.microsoft.com/aspnet/core/fundamentals/target-aspnetcore#use-the-aspnet-core-shared-framework) to the tests instead.

Reply:
Do I need to actually use `IHttpContextAccessor`? Maybe I can demonstrate it just on the `ResilienceContext`?

Reply:
I don't mind - the HTTP context way might be more familiar to people though.

Reply:
I'll add the comment about it in the test.